src/components/invest/investslug/components/TransparentPlayButton.tsx

```diff
@@ -0,0 +1,24 @@
+const TransparentPlayButton = () => {
+  return (
+    <button className='relative flex items-center justify-center w-[193px] h-[193px] rounded-full bg-transparent hover:bg-white hover:bg-opacity-10 transition duration-200'>
+      <div
+        className='absolute inset-0 rounded-full backdrop-blur-[11.90px]'
+        style={{ backgroundColor: 'rgba(19, 19, 19, 0.7)' }}
+      />
+      <svg
```

Comment:
ask Matan to provide you proper svg icon or use PlayIcon from 'lucide-react' package